### PR TITLE
Add credentials

### PR DESCRIPTION
**Pull request by davidlehn**
This is on our fork at the moment.  Can make a PR to main repo after this is looked at.

### CODE REVIEW

**Review by dlongley — APPROVED** [review]
Thanks!

File: contexts/citizenship-v2.jsonld [hunk 1]
```diff
@@ -38,42 +35,169 @@
...
-        "addressRegion": "https://schema.org/addressRegion"
+        "description": "https://schema.org/description",
+        "name": "https://schema.org/name"
       }
     },
 
-    "QuantitativeValue": {
-      "@id": "https://schema.org/QuantitativeValue",
+    "EmploymentAuthorizationPerson": {
+      "@id": "https://w3id.org/citizenship#EmploymentAuthorizationPerson",
```
Comment (msporny) [inline contexts/citizenship-v2.jsonld:91]: This is weird... I get why it's being done... but... weird.

**Review by msporny — APPROVED** [review]

File: contexts/citizenship-v2.jsonld (hunk 1 above)
Comment (davidlehn) [reply contexts/citizenship-v2.jsonld:91]: Yes, it is weird.  I wasn't sure what else to use that fit the pattern here.  Suggestions welcome.

**Comment by davidlehn** [comment]
Closing this in favor of a PR to the ccg repo: https://github.com/w3c-ccg/citizenship-vocab/pull/21

**Pull request closed without merging**